.travis.yml

```diff
@@ -50,52 +55,42 @@ matrix:
 
     include:
 
-        # Do a coverage test in Python 2.
+        # Test egg_info in Python 2
+        - python: 2.7
+          env: ASTROPY_VERSION=lts SETUP_CMD='egg_info'
+
+        # Do a coverage test in Python 3.
         # *** TODO: `python setup.py test --coverage` doesn't work yet for `ginga` ... skipping for now
-        # - python: 2.7
+        # - python: 3.6
         #  env: SETUP_CMD='test --coverage'
 
         # Check for sphinx doc build warnings - we do this first because it
         # may run for a long time
         # *** TODO: The `-w` option doesn't work with `python setup.py build_sphinx` yet for `ginga` ... removing it for now
-        - python: 2.7
+        - python: 3.6
           # env: SETUP_CMD='build_docs -w'
           env: SETUP_CMD='build_docs' PIP_DEPENDENCIES='sphinx_rtd_theme'
 
-        # Try Astropy development version
-        # NOTE: Uncomment this if global version is stable, not dev
-        - python: 2.7
-          env: ASTROPY_VERSION=development
-        - python: 3.5
-          env: ASTROPY_VERSION=development
-
-        # Try Astropy LTS version (disabled for now)
-        #- python: 2.7
-        #  env: ASTROPY_VERSION=lts
-        #- python: 3.5
-        #  env: ASTROPY_VERSION=lts
-
-        # Try older numpy versions
+        # Try older Numpy versions
+        # TODO: Pin Python 2.7 to ASTROPY_VERSION=lts after Astropy v3 release
         - python: 2.7
```

Comment:
Is it fairly simple to set different supporting package requirements for different python versions?  In particular, is is possible to say that, e.g. for python 2.7 it is ok for numpy>=1.7 while for python 3.x we want numpy>=1.12?

Reply:
This particular file only controls Travis CI testing. It does not stop people from installing weird version combo. Theoretically, the python + numpy combo would be resolved upstream already when people install astropy, which is a dependency of Ginga, so we don't have to worry about it? @bsipocz , do you have anything to add?

Reply:
Maybe you can also somehow declare these version requirement in ``setup.py``, but I'm not sure without asking google/SO how to do that conditionally.

Also, for astropy 2.0.x, you can't have numpy 1.7, it's np >=1.9, while 3.0 will be >=1.10.